django_teams/templatetags/teams.py

```diff
@@ -18,20 +19,30 @@ def verbose_name(object):
 
 def load_fragment_template(object):
     ct = ContentType.objects.get_for_model(object)
+
     template_name = 'django_teams/fragments/%s/%s.html' % (ct.app_label, ct.model)
+    print template.loader.get_template(template_name)
```

Comment:
This causes the template load error on joining classrooms with pending approvals